doc/sphinx/proof-engine/proof-handling.rst

```diff
@@ -285,64 +282,56 @@ their own proof modes. The default proof mode used when opening a proof can
 be changed using the following option.
 
 .. opt:: Default Proof Mode @string
-   :name: Default Proof Mode
 
    Select the proof mode to use when starting a proof. Depending on the proof
    mode, various syntactic constructs are allowed when writing an interactive
    proof. The possible option values are listed below.
 
-   - "Classic": this is the default. It activates the |Ltac| language to interact
+   `"Classic"`
+     the default. Activates the |Ltac| language to interact
      with the proof, and also allows vernacular commands.
 
-   - "Noedit": this proof mode only allows vernacular commands. No tactic
+   `"Noedit"`
+     only allows vernacular commands. No tactic
      language is activated at all. This is the default when the prelude is not
      loaded, e.g. through the `-noinit` option for `coqc`.
 
-   - "Ltac2": this proof mode is made available when requiring the Ltac2
-     library, and is set to be the default when it is imported. It allows
-     to use the Ltac2 language, as well as vernacular commands.
+   `"Ltac2"`
+     only allows the Ltac2 language and vernacular commands.
+     Only available and becomes the default after requiring Ltac2.
```

Comment:
Given that all proof modes support vernacular commands, maybe I should say that once above the mode descriptions rather than repeating that for each mode.  Do you think it's likely we'd create a proof mode that doesn't allow vernacular commands?

BTW, do you know if there a proof mode for SSR?  @gares?

Reply:
No, SSR extends the default proof mode with new tactics which you can mix with the vanilla ones.

Reply:
>Do you think it's likely we'd create a proof mode that doesn't allow vernacular commands?

No, it's not. Yes, I think that removing this repeated mention or factoring it would be an improvement.